scripts/ta_bin_to_c.py

```diff
@@ -37,6 +39,16 @@ def get_args():
 
     return parser.parse_args()
 
+def ta_get_flags(ta_f):
+
+    with open(ta_f, 'rb') as f:
+        elffile = ELFFile(f)
+
+        for s in elffile.iter_sections():
+            if s.name == '.ta_head':
```

Comment:
I'm afraid we need to do `get_name(s) == '.ta_head'` here in order to avoid compatibility issues with the `elftools` module. See https://github.com/OP-TEE/optee_os/blob/3.8.0/scripts/gen_tee_bin.py#L65-L72.
I mean, the simple version works for me (Ubuntu 19.10) but it might not work on some distributions.